openpass-js-sdk/src/views/quick-auth-popup-session-check.html

```diff
@@ -0,0 +1,117 @@
+        crossorigin="anonymous" />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.myopenpass.com/openpass-js-sdk/v4/openpass-js-sdk.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+</head>
+
+<body>
+    <main>
+        <div id="openpass-quick-auth-demo"></div>
+        <div class="container p-5">
+            <div class="card">
+                <div class="card-body">
+                    <p class="card-text">
+                        <a href="/">Back to samples list</a>
+                    </p>
+                </div>
+            </div>
+            <div class="card mt-5 invisible" id="result-card">
+                <div class="card-body">
+                    <h5 class="card-title" id="result-title"></h5>
+                    <p class="card-text" id="result-body"></p>
+                </div>
+            </div>
+        </div>
+    </main>
+</body>
+
+
+<script src="http://localhost:8885/sdk"></script>
+<script>
+
+    let authClient = undefined;
+    const resultCard = document.getElementById("result-card");
+    const resultTitle = document.getElementById("result-title");
+    const resultBody = document.getElementById("result-body");
+
+    function popupSuccessCallback(signInResponse) {
+
+        console.log("Popup flow sign in succeeded, email address: " + signInResponse.idToken.email);
+
+        resultTitle.innerText = "Success!";
+        resultBody.innerText = "Signed in as: " + signInResponse.idToken.email;
+        resultCard.classList.add("text-bg-success");
+        resultCard.classList.remove("invisible");
+    }
+
+    function popupFailedCallback(error) {
+
+        console.log("Popup flow sign in failed, error: " + error.message);
+
+        resultTitle.innerText = "Error";
+        resultBody.innerText = "Authentication failed: " + error.message;
+        resultCard.classList.add("text-bg-danger");
+        resultCard.classList.remove("invisible");
+    }
+
+    let quickAuthInitialized = false;
+
+    function visibilityChangedCallback(data) {
+        console.log("Visibility changed: " + data.visibility);
+
+        resultTitle.innerText = "Visibility changed";
+
+        if (quickAuthInitialized) {
+            resultBody.innerText = "Visibility changed to: " + data.visibility;
+            resultCard.classList.add("text-bg-success");
+        } else if (data.visibility) {
+            resultBody.innerText = "Quick Auth is visible because there is an active OpenPass session on this browser";
+            resultCard.classList.add("text-bg-success");
+        } else {
+            resultBody.innerText = "Quick Auth is not visible because there is no active OpenPass session on this browser";
+            resultCard.classList.add("text-bg-danger");
```

Comment:
Maybe warning? It's not really an error so red might be a bit aggressive

Reply:
Done